data/mods/blazemod/blaze_autoweapons_parts.json

```diff
@@ -1,428 +1,337 @@
 [
+  {
+    "abstract": "turret_blaze",
```

Comment:
Adding this allows to reduce the file even more, isn't it?
```
     "color": "dark_gray",
     "broken_color": "dark_gray",
```

Reply:
There are different colors for some turrets, for example `abolter` has `light_gray` colors.

Reply:
Okay, done.

Reply:
Yeaah, diff +2 −282 looks great  :scissors:  :thumbsup: